src/Client.ts

```diff
@@ -96,11 +87,14 @@ export default class Client extends EventEmitter {
     return await stream.publish(this.rid);
   }
 
-  async subscribe(mid: string): Promise<RemoteStream> {
```

Comment:
What do you think of tracking this internally? Add a property that stores tracks by mid when `stream-add` is received. Automatically set it when `subscribe` is called. Then we don't need to expose this info to the consumer (since I dont think it is useful for them?)